docs/content/en/docs/operator-manual/control-plane/installation.md

```diff
@@ -76,43 +76,9 @@ helm install pipecd pipecd/pipecd --version=VERSION --namespace=NAMESPACE \
   --set-file secret.gcsServiceAccount.data=path-to-service-account-file
 ```
 
-#### Using DynamoDB and S3
+Currently, besides `Firestore` PipeCD supports other databases as its datastore such as `MongoDB` and `MySQL`. Also as for filestore, PipeCD supports `AWS S3` and `MINIO` either.
 
-> TBA
-
-#### Using MongoDB and Minio
-
-``` yaml
-apiVersion: "pipecd.dev/v1beta1"
-kind: ControlPlane
-spec:
-  stateKey: {RANDOM_STRING}
-  datastore:
-    type: MONGODB
-    config:
-      url: {YOUR_MONGODB_ADDRESS}
-      database: {YOUR_DATABASE_NAME}
-  filestore:
-    type: MINIO
-    config:
-      endpoint: {YOUR_MINIO_ADDRESS}
-      bucket: {YOUR_BUCKET_NAME}
-      accessKeyFile: /etc/pipecd-secret/minio-access-key
-      secretKeyFile: /etc/pipecd-secret/minio-secret-key
-      autoCreateBucket: true
-```
```

Comment:
This should be kept as an example of using `MongoDB` and `MINIO`.
After completely migrating to `MySQL` we will replace `MongoDB` by `MySQL` too.

Reply:
I get your point 👍 addressed by https://github.com/pipe-cd/pipe/pull/1719/commits/8bc47abb3e28bfe39acad5d5182fff18bb2f8f0c 🙏